src/View/Helper/HtmlHelper.php

```diff
@@ -659,12 +663,12 @@ public function style(array $data, bool $oneLine = true): string
      * - `fullBase` If true the src attribute will get a full address for the image file.
      * - `plugin` False value will prevent parsing path as a plugin
      *
-     * @param string|array $path Path to the image file, relative to the app/webroot/img/ directory.
+     * @param string $path Path to the image file, relative to the app/webroot/img/ directory.
      * @param array $options Array of HTML attributes. See above for special options.
      * @return string completed img tag
      * @link https://book.cakephp.org/3.0/en/views/helpers/html.html#linking-to-images
      */
-    public function image($path, array $options = []): string
+    public function image(string $path, array $options = []): string
```

Comment:
This seems like something that is going to break for people.

Reply:
That is why i made the PR with speaking deprecations in 3.9.